style.css

```diff
@@ -187,35 +187,6 @@ label {
     border-radius: 16px;
   }
 
-  .portfolio-image {
-    min-width: 295px;
-    height: 220px;
-    background: url('images/work1.png');
-    border-radius: 8px;
-    background-size: 100%;
-  }
-
-  .portfolio-image1 {
-    min-width: 295px;
-    height: 220px;
-    background: url('images/work2.png');
-    border-radius: 8px;
-  }
-
-  .portfolio-image2 {
-    min-width: 295px;
-    height: 220px;
-    background: url('images/work3.png');
-    border-radius: 8px;
-  }
-
-  .portfolio-image3 {
-    min-width: 295px;
-    height: 220px;
-    background: url('images/work4.png');
-    border-radius: 8px;
-  }
-
   .work-description {
     display: flex;
     flex-direction: column;
```

Comment:
As requested by the previous reviewer, please let's avoid multiple repeated CSS properties across our stylesheet. It would be great if you can find a way of reducing this duplication of code so as to Follow our list of [best practices for HTML & CSS](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md).

In this case, you could do this with flex:

```CSS
.flex {
  display: flex;
}
```
And then, use it in your HTML elements where you need a flex container: 🤗

```CSS
<div class="flex"> This is a flex container </div>
```